### ISSUE TITLE
Kolejność płatności - osobna karta nie jest dostępna w zmianie pozycji

### ISSUE THREAD

**Issue by Triloworld** [PayU-EMEA/plugin_prestashop#306]
Ta opcja:
<img width="437" alt="Zrzut ekranu 2022-11-29 o 18 16 05" src="https://user-images.githubusercontent.com/2671726/204597462-cb438c03-87a8-4952-b8ec-5288c5e40595.png">

Ten kod odpowiada za dodanie karty i PayU: https://github.com/PayU-EMEA/plugin_prestashop/blob/fac720a1fcac10433f0ce3fc2c22e27e26ccaab0/payu/payu.php#L776-L991

Na tej liście jest możliwość ustalania pozycji sposobów płatności:
<img width="1051" alt="Zrzut ekranu 2022-11-29 o 18 18 12" src="https://user-images.githubusercontent.com/2671726/204597912-c77f5a08-30e2-4891-a7b5-fcb66dbaa103.png">

W tej chwili metoda płatności kartą jest ZAWSZE nad metodą PayU jako że kod deklaruje ją powyżej standardowej metody a nie poniżej.

W tej chwili ta opcja nie jest intuicyjna, nie działa z osobną metodą płatności kartą, powoduje błędy a także w dalszej kolejności uniemożliwia mieszanie pozycji metod płatności z zewnętrznymi/wbudowanymi dostawcami:
<img width="1059" alt="Zrzut ekranu 2022-11-29 o 18 21 05" src="https://user-images.githubusercontent.com/2671726/204598503-8c856780-e07d-4317-8fb8-61a9a725cd5e.png">

Rozwiązanie błędu jest umożliwienie by w ramach natywnego sposobu wyboru płatności można było zarządzać z poziomu 
`Wygląd -> Pozycje -> paymentOptions` wszystkich dostępnych metod dla klienta. By były one przypinane po pobraniu ustawień z panelu po aktywacji prawidłowej modułu i umożliwiały pełną konfigurację w ramach natywnych funkcjonalności prestashop.
Poza możliwością łatwego odczepiania/ustalania kolejności potrzebny byłby przycisk do resetowania tych zmian w pozycjach.

**Comment by regdos** [comment]
Zgadza się, że w ramach konfiguracji zaczepu `paymentOptions` można zmieniać kolejność. Niestety Plugin może zarejestrować tylko jeden zaczep danego rodzaju więc nie ma możliwości żeby każda wydzielona metoda (karta, blik, raty) miała swój zaczep co skutkuje tym, że metody PayU muszą być zgrupowane razem. 
Zgadzam się, że nazwa opcji `Kolejność metod płatności` może być nieprecyzyjna ale w dokumentacji https://github.com/PayU-EMEA/plugin_prestashop#spos%C3%B3b-integracji jest informacja, że dotyczy tylko listy banków. 

Kolejności metod w ramach wszystkich dostępnych nie można w tej chwili ustawić ale nawet jeżeli to dorobimy to będzie to ograniczenie, że nie mogą być mieszane z innymi metodami.
